lite-locals.tf

```diff
@@ -8,17 +66,80 @@ locals {
 
   owner_name_initials = lower(join("",[for x in split(" ",local.owner.name): substr(x,0,1)]))
 
-  tags = var.extra_tags
+  /*metadata = {
```

Comment:
Please delete commented-out code.

Reply:
fixed

Reply:
It is not fixed.  See line 94, below, as a simple example.